app/renderer/js/main.js

```diff
@@ -572,6 +600,7 @@ window.onload = () => {
 	serverManagerView.init();
 
   window.serverManagerView = serverManagerView;
+  window.FunctionalTab = FunctionalTab;
```

Comment:
These here, are just for testing, provides a great way see what happens underneath the hood using dev-tools.

Just leaving a comment so I know to remove them when cleaning up.